extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -82,6 +82,57 @@ private static function renderGallery( $media ) {
 		);
 	}
 
+	public static function getDataAttrsForImage( $media ) {
+		$dataAttrs = [
```

Comment:
that actually should be an object right? :)

Reply:
or use that method in a method for getting data attrs for gallery to avoid repetion

Reply:
done